package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "start": "webpack-dev-server --config test/screenshot/webpack.config.js --content-base test/screenshot",
     "stop": "./test/screenshot/stop.sh",
-    "capture": "./node_modules/.bin/mocha --compilers js:babel-core/register --ui tdd test/screenshot/capture-suite.js",
+    "capture": "COMMIT_HASH=$(git rev-parse --short HEAD) BRANCH_NAME=$(git rev-parse --abbrev-ref HEAD) mocha --compilers js:babel-core/register --ui tdd --timeout 30000 test/screenshot/capture-suite.js",
```

Comment:
If we want this to run on Windows, we'll need to use `cross-env` (see MDC Web's package.json for examples).

Also, keep in mind that there may be unstaged changes in the working directory that won't be reflected in `COMMIT_HASH`. This may end up biting us down the road if we forget that fact and assume that screenshots with a particular commit hash are from that exact commit with no other changes.